lib/ret_web/router.ex

```diff
@@ -199,6 +199,7 @@ defmodule RetWeb.Router do
 
     scope "/v1", as: :api_internal_v1 do
       get("/presence", ApiInternal.V1.PresenceController, :show)
+      get("/presence/daily_max", ApiInternal.V1.PresenceController, :daily_max)
```

Comment:
nit: Not a big deal, since this is an internal API, but `daily_max` implies a fixed period, when the API actually accepts any range. So maybe just call it `/presence/max`, or `/presence/range_max`, or something.

Same for the function name in PresenceController.